public/app/plugins/datasource/azuremonitor/components/TracesQueryEditor/Filter.tsx

```diff
@@ -78,6 +77,7 @@ const getTraceProperties = async (
   | summarize make_list(pack_all()));
   print properties = bag_pack("${property}", ${property});`;
 
+  const range = (getTemplateSrv() as any).timeRange;
```

Comment:
Can we get this from the query properties instead where it might be typed?

Reply:
This is putting it in the query props right? I think I am missing something